tests/cases/fourslash/quickInfoJSDocBackticks.ts

```diff
@@ -11,15 +11,18 @@
 //// * @param {string} x hi there `@param`
 //// * @param {string} y hi there `@ * param
 //// *                   this is the margin
+//// * @param {string} z OOPS, unclosed backtick!
 //// */
-////export function f(x, y) {
-////    return x/*x*/ + y/*y*/
+////export function f(x, y, z) {
+////    return x/*x*/ + y/*y*/ + z/*z*/
 ////}
 ////f/*f*/
 
 goTo.marker("f");
-verify.quickInfoIs("function f(x: string, y: string): string", "`@param` initial at-param is OK in title comment");
+verify.quickInfoIs("function f(x: string, y: string, z: any): string", "`@param` initial at-param is OK in title comment");
 goTo.marker("x");
 verify.quickInfoIs("(parameter) x: string", "hi there `@param`");
 goTo.marker("y");
-verify.quickInfoIs("(parameter) y: string", "hi there `@ * param\nthis is the margin");
+verify.quickInfoIs("(parameter) y: string", "hi there `@ * param\nthis is the margin\n@param {string} z OOPS, unclosed backtick!");
```

Comment:
This isn't correct. Markdown won't detect this as a code span because there's no closing backtick, so it will treat the backtick as a literal character.